src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
```

Comment:
That's not the case if it was running on Mono -- it would be a no-op there. (Why would there be such code? Who knows .. copy paste). As I say though, since there may be zero usage, fortunately we don't have this dilemma. :) 

If you mean we throw PNSE from all of CAS, that won't fly as nothing in Mono would work. :)

---
In reply to: [70846341](https://github.com/dotnet/corefx/pull/10049#discussion_r70846341) [](ancestors = 70846341)